astropy/table/table.py

```diff
@@ -1903,6 +1903,52 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def remove_na(self):
+        '''
+        Remove rows from a table that has N/A.
+        For masked tables only
+
+        Parameters
+        ----------
+        Masked Table
+
+        Examples
+        --------
+        Create a table with three columns 'a', 'b' and 'c'::
+
+            >>> weather_data = """
+            ...      day,temp,type
+            ...      ,35,rainy
+            ...      Tues,55,sunny
+            ...      Wed,31,snowy
```

Comment:
You can drop the one (unmasked) row of data from the example to make it a bit shorter without losing information.